platform/ui/src/components/Select/Select.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import ReactSelect from 'react-select';
+
+import './Select.css';
+
+const Select = ({
+  autoFocus,
+  className,
+  classNamePrefix,
+  isDisabled,
+  isMulti,
+  isSearchable,
+  name,
+  onChange,
+  options,
+  placeholder,
+  noOptionsMessage,
+  value,
+}) => {
+  return (
+    <ReactSelect
+      autoFocus={autoFocus}
+      className={classnames(
+        className,
+        'flex flex-col flex-1 hover:cursor-pointer mt-2 customSelect__wrapper'
+      )}
+      classNamePrefix="customSelect"
+      isDisabled={isDisabled}
+      isMulti={isMulti}
```

Comment:
When `isMulti` is enabled, it's good to check how the layout should behave. We could share with Dan asking his opinion.

Temporary preview:

![image](https://user-images.githubusercontent.com/1905961/77115331-e4dc0e00-6a0c-11ea-96b5-2211a79d5f04.png)

Reply:
Thast a good point. 
I haven't asked before cause our use of it does not hit the mult select feature yet.